stable/datacube-data/templates/deployment.yaml

```diff
@@ -65,6 +86,12 @@ spec:
             value: {{ $value | quote }}
 {{- end }}
 {{- end }}
+          # Mount the config volume
+          volumeMounts:
+{{- if or .Values.wmsConfig.configMap .Values.wmsConfig.image }}
```

Comment:
humm.. condition seems wrong if you reading value for `{{ .Values.wmsConfig.path }}` ??

Reply:
It is a straight copy-paste. Tested with `helm template .` to render the template. Can review some more.